google/cloud/aiplatform/training_jobs.py

```diff
@@ -987,6 +995,80 @@ def __init__(
                 send health check requests to the container, and which must be supported
                 by it. If not specified a standard HTTP path will be used by AI
                 Platform.
+            model_serving_container_command: Optional[Sequence[str]]=None,
```

Comment:
Formatting on the type signatures on these arguments.

Reply:
Sorry, what do you mean?

Reply:
For example:
`model_serving_container_command: Optional[Sequence[str]]=None,`

should be:

`model_serving_container_command (Sequence[str]):`

Reply:
Oh ok, I didn't see that it was the comments you were commenting on.